web/static/js/channels/repository_channel.js

```diff
@@ -3,6 +3,7 @@ export default function repositoryChannelHandler (socket, userId) {
   let channel = socket.channel(`repository:list:${userId}`, {})
   channel.join()
     .receive('ok', resp => {
+      console.log("Joined repository list channel")
```

Comment:
This looks like something that shouldn't be committed - although I guess any user of Pullhub is going to have a `console` object :)